shipyard_blueprints/postgresql/shipyard_postgresql/errors/exceptions.py

```diff
@@ -0,0 +1,23 @@
+from logging import error
+from shipyard_templates import Database, ExitCodeException
+
+EXIT_CODE_UPLOAD_ERROR = 101
+EXIT_CODE_CHUNK_DOWNLOAD_ERROR = 102
+
+
+class InvalidCredentials(ExitCodeException):
```

Comment:
If you want to follow the benefit of adding common exceptions to the abstract class this should be move to the database class